docs/application/tizen-studio/common-tools/device-manager.md

```diff
@@ -37,36 +37,46 @@ To use the Device Manager with a specific device, you must connect the device ov
 <a name="connecting"></a>
 ## Connecting Devices
 
-The Device Manager shows both local devices connected to the system and any configured remote devices:
+Device Manager shows both local devices connected to the system and any configured remote devices:
 
 -   Only connected local devices (both physical and virtual devices) are listed. Disconnected devices automatically disappear from the list. Therefore, the **Connection** switch is disabled.
 -   Remote devices are listed even if the connection is lost. You can turn the connection on and off with the **Connection** switch.
 
-**Figure: Connected devices in the Device Manager**
+**Figure: Connected devices in Device Manager**
 
-![Connected devices in the Device Manager](./media/device_manager_connected_devices.png)
+![Connected devices in Device Manager](./media/device_manager_connected_devices.png)
 
 Connecting a device to a computer depends on the device type:
 
 -   Physical device
 
-    When you connect a local physical device with a USB cable, it automatically appears in the Device Manager.
+    When you connect a local physical device with a USB cable, it automatically appears in Device Manager.
 
 - Virtual device
 
-    To connect a virtual device, click the **Emulator Manager** button (![Emulator Manager](./media/device_manager_emulator_manager_icon.png)), select the virtual device, and click **Launch**.
+    To connect a virtual device, click **Emulator Manager** ![Emulator Manager](./media/device_manager_emulator_manager_icon.png), select the virtual device, and click **Launch**.
```

Comment:
According to Google Style Guide, I have modified the sentence.
For more information, see https://developers.google.com/style/ui-elements.

As I have changed line number 66, "In the **Device Manager** window that appears,  click **Remote Device Manager** ![Remote Device Manager](./media/device_manager_remote_device_manager_icon.png). ". To maintain consistency, I used the same style and removed "button" wherever applicable throughout the page.

Please let me know your valuable opinion on the same.

Reply:
@Reni-90 To get the consistency through the pages, please remember that you should check other pages also, not only inside of a page. If we decide to reference Google Style Guide not to emphasize "button", we'd better apply the same style to other pages.

Please share "not to focus on the UI element but on the task" with other reviewers, so that they will check this point also when other pages are under review. As there is another PR for Tizen Studio revamp, I think it is good chance to fix to use this style.

Reply:
@mijin85cho 
Thank you for your opinion.
I understand the consistency issue. 
As I was changing this page, I thought of fixing all the issues. I will change it back to the old style throughout the page and will inform other reviewers also. 
Yes, we can fix these style issues in the  PR for Tizen Studio revamp.

Reply:
As per our discussion, I have retained the new style and informed the Tizen Studio revamp owner to update the same style to other pages.